webapp/src/components/onboarding/EditorTour.tsx

```diff
@@ -4,22 +4,23 @@ import { Location, TeachingBubble, TargetContent } from "../../../../react-commo
 
 const Simulator: TargetContent = {
     title: lf("Micro:bit Simulator"),
-    description: lf("The simulator shows what your program will look like running on a micro:bit."),
+    description: lf("See what your code looks like running on a micro:bit!"),
     targetQuery: "#boardview",
     location: Location.Right,
 };
 
 const Toolbox: TargetContent = {
     title: lf("Toolbox"),
-    description: lf("The toolbox is organized into different categories with blocks of code you can drag out and use in your program. You can search for specific blocks, open the extensions gallery for more blocks, or click Advanced to expand the toolbox categories shown."),
+    description: lf("Drag out blocks of code from the Toolbox categories into the Workspace."),
```

Comment:
nit: Something like "Drag blocks of code out of the Toolbox categories into the Workspace" feels a little cleaner to me, but I think it's just a matter of personal preference.